pkg/driver/node.go

```diff
@@ -386,7 +386,7 @@ func (d *nodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVo
 	klog.V(4).Infof("NodeGetVolumeStats: called with args %+v", *req)
 	var resp *csi.NodeGetVolumeStatsResponse
 
-	if req == nil || req.VolumeId == "" {
+	if req.VolumeId == "" {
```

Comment:
this still is not resolving the issue, I feel can be written as

```suggestion
	if req != nil && req.VolumeId == "" {
```

Reply:
This doesn't fix the issue. However, I got the right fix.

Reply:
Please check the current changes